src/mode_RPiHQ_mean.cpp

```diff
@@ -180,7 +183,7 @@ if (0)
 			break;
 	}
 		 
-	Log(3, "  > exposure: %.3f sec, mean: %1.4f, mean_value: %1.4f, diff: %1.4f\n", ExposureTime_s, mean, currentModeMeanSetting.mean_value, (currentModeMeanSetting.mean_value - mean));
+	Log(3, "  > exposure: %.3f sec, mean: %1.3f, mean_value: %1.3f, diff: %'1.3f\n", ExposureTime_s, mean, currentModeMeanSetting.mean_value, (currentModeMeanSetting.mean_value - mean));
```

Comment:
?

Reply:
@AndreasLMeg   I made all the mean precisions 1.3 to be consistent with what's displayed on the image, so it's easier to associate a log entry with an image.  The apostrophe is supposed to add thousands separators to number, e.g, in the US: 1,000 for one-thousand and 1.000 for you.  Unfortunately it doesn't work for any fractional part, so in the US: 0.123456 whereas what I want is 0.123,456